client/blocks/reader-post-card/style.scss

```diff
@@ -212,6 +240,11 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	}
 }
 
+.reader-post-card__meta-separator {
```

Comment:
I considered that, but I was concerned about accessibility:

http://tink.uk/accessibility-support-for-css-generated-content/

The comma separator is part of the content, and we'd want a pause if it was being read by a screen reader, so I thought it best to include it directly rather than using `::after`.